linux_os/guide/services/obsolete/inetd_and_xinetd/configure_etc_hosts_deny/rule.yml

```diff
@@ -10,6 +10,10 @@ description: |-
     The following line in the file ensures that access to services supporting this mechanism is denied to any clients
     not mentioned in <tt>/etc/hosts.allow</tt>:
     <pre>ALL: ALL</pre>
+    It is advised to inspect available network services which might be affected by modification of file mentioned above prior to performing the remediation of this rule.
+    If there exist services which might be affected and access to them should not be blocked,
+    modify the <tt>/etc/hosts.deny</tt> file appropriately before performing the remediation.
```

Comment:
Shouldn't `/etc/hosts.allow` be the file modified to grant access?

Suggestion: `Every service that is essential for your organization should be allowed access before applying remediation for this rule.
Edit <tt>/etc/hosts.allow</tt> and add the allowed services and hosts.`